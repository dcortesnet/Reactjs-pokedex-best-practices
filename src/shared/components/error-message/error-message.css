.error__container {
  padding-block-start: 50px;
  width: 100%;
  height: 100%;
  display: flex;
  overflow: hidden;
  justify-content: center;
  align-items: center;
  color: red;
}
