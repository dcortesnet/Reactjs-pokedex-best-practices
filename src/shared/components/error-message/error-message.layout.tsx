import React from "react";
import "./error-message.css";

export const ErrorMessageLayout = () => {
  return (
    <div className="error__container">
      <div className="error">
        <a href="https://fontmeme.com/es/fuente-pokemon/">
          <img
            src="https://fontmeme.com/permalink/210609/22d6efb960600fa3e82598cb3b651b33.png"
            alt="fuente-pokemon"
          />
        </a>
      </div>
    </div>
  );
};
