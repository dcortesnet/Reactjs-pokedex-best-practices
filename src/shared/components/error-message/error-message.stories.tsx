import { Story, Meta } from "@storybook/react";
import { ErrorMessageLayout } from "./error-message.layout";

export default {
  title: "Shared/components/ErrorMessage",
  component: ErrorMessageLayout,
} as Meta;

export const Template: Story<any> = (args) => <ErrorMessageLayout />;
