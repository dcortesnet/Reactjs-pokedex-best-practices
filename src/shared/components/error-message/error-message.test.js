import { ErrorMessageLayout } from "./error-message.layout";
import React from "react";
import Enzyme, { shallow, mount } from "enzyme";
import Adapter from '@wojtekmaj/enzyme-adapter-react-17';
import toJson from "enzyme-to-json";

Enzyme.configure({ adapter: new Adapter() });

describe("ErrorMessageLayout", () => {
  it("Renders correctly", () => {
    const r = shallow(<ErrorMessageLayout></ErrorMessageLayout>);

    expect(toJson(r)).toMatchSnapshot();
  })
});
