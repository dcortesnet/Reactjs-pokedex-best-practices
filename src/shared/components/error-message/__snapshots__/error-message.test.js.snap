// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ErrorMessageLayout Renders correctly 1`] = `
<div
  className="error__container"
>
  <div
    className="error"
  >
    <a
      href="https://fontmeme.com/es/fuente-pokemon/"
    >
      <img
        alt="fuente-pokemon"
        src="https://fontmeme.com/permalink/210609/22d6efb960600fa3e82598cb3b651b33.png"
      />
    </a>
  </div>
</div>
`;
